pandas/tests/window/test_rolling.py

```diff
@@ -1424,3 +1424,11 @@ def test_rolling_zero_window():
     result = s.rolling(0).min()
     expected = Series([np.nan])
     tm.assert_series_equal(result, expected)
+
+
+def test_rolling_float_dtype(float_dtype):
```

Comment:
this worked in 1.1.5, see https://github.com/pandas-dev/pandas/issues/42452#issuecomment-891732335

we may want to consider backporting this fix.